Automattic/wc-calypso-bridge
# Reduce Facebook and Mailchimp task times

Closes #347.

This PR reduces the task times for Facebook and Mailchimp per the feedback thread.

To Test:
* Load up the setup task list and verify both Facebook and Mailchimp tasks are 5 minutes.

<img width="1322" alt="screen shot 2018-11-28 at 3 09 49 pm" src="https://user-images.githubusercontent.com/689165/49179308-0862dd00-f320-11e8-9ff9-9810c79aee72.png">
